eslint.config.js

```diff
@@ -0,0 +1,190 @@
+import { includeIgnoreFile } from "@eslint/compat";
+import eslint from "@eslint/js";
+import pluginVitest from "@vitest/eslint-plugin";
+import skipFormattingConfig from "@vue/eslint-config-prettier/skip-formatting";
+import vueTsEslintConfig from "@vue/eslint-config-typescript";
+import playwrightEslintConfig from "eslint-plugin-playwright";
+import vue from "eslint-plugin-vue";
+import vueScopedCss from "eslint-plugin-vue-scoped-css";
+import vueA11y from "eslint-plugin-vuejs-accessibility";
+import { fileURLToPath } from "node:url";
+import tsEslint from "typescript-eslint";
+import sitOnyx from "./packages/eslint-plugin/src/index.cjs";
+
+/**
+ * General Vue and TypeScript lint rules without typechecked lint rules.
+ * Linting with typechecking has been disabled by default to improve lint performance.
+ * Enabling the projectService and typechecked lint rules increases linting times drastically (from 20s to about 100s for the whole project)
+ */
+const generalVueTsConfig = {
+  name: "general-vue-ts",
+  files: ["**/*.{js,jsx,ts,tsx,vue}"],
+  extends: [
```

Comment:
Extends seems to be a removed feature which only works here since we use `tsEslint.config()` where I am not sure of we should use it and if/how long it will still be supported in the future

Reply:
`extends` is a quality-of-life feature of the `typescript-eslint` package and for neither there is any indication that they will be deprecated.
The `extends` here is not intended and not used as a backwards compatiblity feature.
Instead we use it to improve readability and reduce duplications.
For all the reasons I think we are good here.